.github/workflows/build-devcontainer.yaml

```diff
@@ -41,17 +49,11 @@ jobs:
           file: .devcontainer/base-image/Dockerfile
           tags: ${{ steps.meta.outputs.tags }}
           platforms: linux/amd64, linux/arm64
-          push:
-            # If it's a push to main, or a tag, or explicitly called, push the image to the registry.
-            # Don't if it's on a schedule.
-            ${{ (github.ref == 'refs/heads/main' && github.event_name == 'push')
-            || startsWith(github.ref, 'refs/tags/') || github.event_name ==
-            'workflow_dispatch' }}
+          push: ${{ inputs.publish == 'true' }}
           cache-from: |
             ${{ steps.meta.outputs.tags }}
             type=gha
```

Comment:
This line can be removed.

Reply:
I thought it would be easier to revert the `cache-to` change by keeping this, and the comment below somewhat explains the current incongruity between them.

Will add a comment (but feel free to remove this if I'm not making sense :) )

Reply:
Hmmm, given the fact that communicating with gha can cause problems, wouldn't leaving gha here cause a CI failure?

Reply:
It seems to only be on the push (this run was passing until it was canceled...)